mage_ai/orchestration/db/__init__.py

```diff
@@ -32,7 +32,7 @@
     else:
         if is_test():
             db_connection_url = f'sqlite:///{TEST_DB}'
-        elif os.path.exists('mage_ai/orchestration/db/'):
+        elif os.path.exists(os.path.join('mage_ai', 'orchestration', 'db')):
             # For local dev environment
             db_connection_url = 'sqlite:///mage_ai/orchestration/db/mage-ai.db'
```

Comment:
we should probably change this as well

Reply:
the sqlite path is using a file uri, which i don't like, but should probably be fine: https://en.wikipedia.org/wiki/File_URI_scheme and e.g. https://docs.sqlalchemy.org/en/20/core/engines.html#sqlite